From 2acc5c3278354f56c70f9f48e66eb0203d5e8370 Mon Sep 17 00:00:00 2001
From: Oleksandr Natalenko <oleksandr@natalenko.name>
Date: Sat, 17 Nov 2018 15:43:09 +0100
Subject: [PATCH] lddtree.py: use errno properly

os.errno interface is removed [1], thus should not be used. In fact, it
is not used in other places of lddtree.py, so fix this one too, because
it breaks --copy-to-tree on systems with Python 3.7.

[1] https://bugs.python.org/issue33666

Bug: https://bugs.gentoo.org/671522
Signed-off-by: Oleksandr Natalenko <oleksandr@natalenko.name>
Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
---
 lddtree.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lddtree.py
+++ b/lddtree.py
@@ -122,7 +122,7 @@ def makedirs(path):
   try:
     os.makedirs(path)
   except OSError as e:
-    if e.errno != os.errno.EEXIST:
+    if e.errno != errno.EEXIST:
       raise
 
 
-- 
2.19.2

